airflow/dags/deploy_dbt_docs/sync_dbt_docs_to_metabase.yml

```diff
@@ -0,0 +1,62 @@
+operator: 'operators.PodOperator'
+name: 'sync-dbt-docs-to-metabase'
+image: 'ghcr.io/cal-itp/data-infra/warehouse:{{ image_tag() }}'
+
+cmds:
+  - python3
+arguments:
+  - '/app/scripts/run_and_upload.py'
+  - 'run'
+  - '--no-dbt-run'
+  - '--dbt-docs'
+  - '--sync-metabase'
+
+is_delete_operator_pod: true
+get_logs: true
+is_gke: true
+pod_location: us-west1
+cluster_name: data-infra-apps
+namespace: airflow-jobs
+priority_class_name: dbt-high-priority
+
+env_vars:
+  AIRFLOW_ENV: "{{ env_var('AIRFLOW_ENV') }}"
+  CALITP_BUCKET__DBT_ARTIFACTS: "{{ env_var('CALITP_BUCKET__DBT_ARTIFACTS') }}"
+  BIGQUERY_KEYFILE_LOCATION: /secrets/jobs-data/service_account.json
+  DBT_PROJECT_DIR: /app
+  DBT_PROFILE_DIR: /app
+  DBT_TARGET: "{{ env_var('DBT_TARGET') }}"
+  NETLIFY_SITE_ID: cal-itp-dbt-docs
+
+secrets:
+  - deploy_type: volume
+    deploy_target: /secrets/jobs-data/
+    secret: jobs-data
+    key: service-account.json
+  - deploy_type: env
+    deploy_target: METABASE_USER
+    secret: jobs-data
+    key: metabase-user
+  - deploy_type: env
+    deploy_target: METABASE_PASSWORD
+    secret: jobs-data
+    key: metabase-password
+  - deploy_type: env
```

Comment:
do you need the netlify auth in this task?

Reply:
good catch, will remove